feature_creation_scripts/antarctic_ocean_regions/antarctic_ocean_regions.py

```diff
@@ -238,13 +294,15 @@ def main():
 
     fcEW = split_rectangle(
         lon0=-20., lon1=25., lat0=-80., lat1=-55., name='Eastern Weddell Sea',
```

Comment:
```suggestion
        lon0=-20., lon1=25., lat0=-80., lat1=-62., name='Eastern Weddell Sea',
```
@milenaveneziani and @xylar Just to make sure I'm understanding you correctly, you want the northern boundary of the Eastern Weddell to be lowered from -55 degS to -62 degS as shown above?

Reply:
@cbegeman, that seems like what @milenaveneziani is suggesting here but it seems counter to the discussion we had in #104.  So it would be better if we sort it out.  If we want the northern boundary of the Eastern Weddell to be further north but also to match up with the Western Weddell we could make the Eastern Weddell a more complicated polygon as well, now that @cbegeman has added the capability.

Reply:
And I presume that the more complicated polygon would be in order to match the extent of the Weddell Gyre in observations.